src/parser/jobs/blm/modules/RotationWatchdog.tsx

```diff
@@ -347,6 +352,12 @@ export class RotationWatchdog extends RestartWindow {
 			// Let the player rush the Despair if they need to before a downtime/end of fight
 			if (windowMetadata.finalOrDowntime) { return window.data.filter(event => event.action.id === this.data.actions.FIRE_IV.id).length }
 
+			// Windows begun by Manafont by definition should contain the full 6 F4s
+			if (window.data[0].action.id === this.data.actions.MANAFONT.id) {
+				windowMetadata.expectedFire4s = MAX_POSSIBLE_FIRE4
+				return MAX_POSSIBLE_FIRE4
+			}
+
 			// Start off with the baseline assumption they've reached full MP in Umbral Ice, since MP events from the log source aren't reliably timed
 			adjustment = NO_UH_EXPECTED_FIRE4
```

Comment:
_Nit:_ Seeing this in passing, but since now MP generation is deterministic based on casts, can we do without this fudge factor?

Reply:
Potentially, but the opener window is still going to be susceptible to not knowing precisely how much MP was left, in things like 24 mans, if the player didn't umbral soul their way to full MP. Can put it on the future/make things better list, but I don't think it's worth holding up support on.